yarn-project/ethereum/src/test/start_anvil.ts

```diff
@@ -34,6 +34,7 @@ export async function startAnvil(
         blockTime: opts.l1BlockTime,
         stopTimeout: 1000,
         accounts: opts.accounts ?? 20,
+        gasLimit: 45_000_000n,
```

Comment:
The default anvil gas limit is 30M. Our reth devnet runs with 100M gas limit. Sepolia's limit is 60M